samples/sample_encode/src/pipeline_encode.cpp

```diff
@@ -876,10 +877,18 @@ mfxStatus CEncodingPipeline::AllocFrames()
     MSDK_ZERO_MEMORY(VppRequest[0]);
     MSDK_ZERO_MEMORY(VppRequest[1]);
 
+    mfxU16 initialTargetKbps = m_mfxEncParams.mfx.TargetKbps;
+    auto co2 = m_mfxEncParams.GetExtBuffer<mfxExtCodingOption2>();
+
     // Querying encoder
     sts = GetFirstEncoder()->Query(&m_mfxEncParams, &m_mfxEncParams);
     MSDK_CHECK_STATUS(sts, "Query (for encoder) failed");
 
+    if (co2->BitrateLimit != MFX_CODINGOPTION_OFF && initialTargetKbps != m_mfxEncParams.mfx.TargetKbps)
+    {
+        msdk_printf(MSDK_STRING("WARNING: Target bitrate was changed from %d kbps to %d kbps because -BitrateLimit:on option specified.\n"), initialTargetKbps, m_mfxEncParams.mfx.TargetKbps);
```

Comment:
I would put just that:
"WARNING: -BitrateLimit:on: target bitrate was changed from %d kbps to %d kbps"

Reply:
okay, changed